features/built_in_matchers/any.feature

```diff
@@ -0,0 +1,61 @@
+Feature: `any` matcher
+
+  Use the `any` matcher to specify that a collection's objects any pass an expected matcher.
+  This works on any enumerable object.
+
+    ```ruby
+    expect([1, 4, 5]).to any( be_odd )
+    expect([1, 3, 'a']).to any( be_an(Integer) )
+    expect([1, 3, 11]).to any( be < 10 )
```

Comment:
I think we'd want to add and use aliases here, e.g. `have_a_member` or similar more readable examples